Sources/BranchSDK/Branch.m

```diff
@@ -1922,11 +1923,21 @@ - (void)initSafetyCheck {
 
 - (void)initUserSessionAndCallCallback:(BOOL)callCallback sceneIdentifier:(NSString *)sceneIdentifier urlString:(NSString *)urlString {
     
-    // ignore lifecycle calls while waiting for a plugin runtime.
     @synchronized (self) {
         if (self.deferInitForPluginRuntime) {
-            [[BranchLogger shared] logDebug:@"Branch init is deferred, ignoring init call."];
+            if (urlString) {
```

Comment:
Does this impact the ability to enqueue v2 events prior to init? Or will they be dropped still

Reply:
No change from existing behavior. Though we may wish to evaluate if the existing behavior is acceptable in another ticket. SDK-2340